recipes-devtools/rust/rust-llvm_1.24.0.bb

```diff
@@ -0,0 +1,16 @@
+require rust-source-${PV}.inc
+require rust-llvm.inc
+
+LIC_FILES_CHKSUM = "file://LICENSE.TXT;md5=e825e017edc35cfd58e26116e5251771"
+
+do_install_prepend () {
+	# the install does a sed on this without installing the file
```

Comment:
If we don't need it, can't we patch the package? It would be easier to follow up with updates.

Reply:
To me, it is going to be carrying a patch forward vs doing the remove in after the install stage. Neither really seems worse one way or the other to me; I kept the remove after the install stage to be consistent with previous versions of the recipe.

Reply:
Makes sense.